lib/src/components/HalTable.jsx

```diff
@@ -109,24 +106,26 @@ const HalTable = ({ collectionUrl, asyncHeadersHandler, headers, columns, itemsP
     <DxcHALTableContainer>
```

Comment:
Maybe to rename this `DxcHALTableContainer`? We are not using the word `HAL` in uppercase anymore and I would keep the word `Dxc` for the components of `Halstack React`. Something like `HalTableContainer` I think would be better.

Reply:
I agree. Done!